docs/pipeline-configuration-files/provisioning.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Provisioning'
+sidebar_position: 1
+---
+
+## Pipelines Directory
+
+When Conduit starts, it will search for all files ending with `.yml` or `.yaml` in folder `./pipelines` and its
+sub-folders. It will parse the configuration files and provision the pipelines defined in them.
+
+If you want Conduit to search a different directory, or want to provision a single file, you can specify the CLI flag
+`pipelines.path` and point to your file or directory:
+
+```shell
+./conduit -pipelines.path /path/to/my-directory
+```
+
+If the directory does not exist, Conduit will fail with the error `"pipelines.path" config value is invalid`.
+
+## Ignored errors
+
+Conduit will try to provision all pipelines regardless of the errors it may encounter while going through the
+configuration files. Invalid configuration files will be ignored (e.g. files that miss a mandatory field or have an
+invalid configuration value) and won't impact the provisioning of valid configuration files. Conduit will log a warning
+for every error it encounters during this process.
+
+You need to be careful when choosing IDs for your pipelines, connectors and processors. Each pipeline needs to get a
```

Comment:
Feels like this deserves it's own heading rather than being buried under "ignored errors." It is a pretty important detail.